### PR TITLE
adding role for importing the demo course

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/edx-east/edx_continuous_integration.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@
     - supervisor
     - mongo
     - edxapp
+    - role: demo
+      tags: ['demo']
```
Comment (feanil) [inline playbooks/edx-east/edx_continuous_integration.yml:23]: What is this tag for?

**Comment by e0d** [comment]
+1

File: playbooks/edx-east/edx_continuous_integration.yml (hunk 1 above)
Comment (feanil) [inline playbooks/edx-east/edx_continuous_integration.yml:23]: talked to John, makes sense for limiting playbook runs to just the demo role.

**Comment by feanil** [comment]
looks good.

**Pull request merged**